src/server/StudioManagementSystem.Infrastructure/Interfaces/Data/IGroupApplicationRepository.cs

```diff
@@ -7,7 +7,8 @@ public interface IGroupApplicationRepository
 {
     Task<bool> AddGroupApplication(InvitationDto dto, CancellationToken ct);
 
-    Task<List<GroupApplication>> GetGroupApplicationAsync(Guid groupId, CancellationToken ct);
+    Task<List<GroupApplication>> GetGroupApplicationsAsync(Guid groupId, CancellationToken ct);
+    Task<bool> RejectGroupApplication(RemoveInvitationDto dto, CancellationToken ct);
```

Comment:
The repo is where we manage data-layer logic (`aka. CRUD à la carte`). Rejection is a biz level concept. See the `Manager` services I've started building that handle biz logic.

> Think of a manager as a business person who manages a product, but the repo as an warehouse worked moving inventory

Reply:
ie.,

```suggestion
    // IGroupApplicationRepository
    Task<bool> RemoveGroupApplicationsAsync(List<Guid> ids, CancellationToken ct);
    
    // IGroupManager (I think this already exists)
    Task<bool> RejectGroupApplicationsAsync(List<RemoveInvitationDto> dto, CancellationToken ct);
```